phidgets_api/src/motor.cpp

```diff
@@ -215,7 +240,10 @@ void Motor::dutyCycleChangeHandler(double duty_cycle) const
 
 void Motor::backEMFChangeHandler(double back_emf) const
 {
-    back_emf_change_handler_(channel_, back_emf);
+    if (back_emf_sensing_supported_)
```

Comment:
This shouldn't be necessary; if back EMF is not supported, then we would have failed to install the change handler during construction.  So I think you can just remove this.